Replace scsi_scan_target (unexported by kernel < 2.6.12) by scsi_scan_host

Signed-off-by: Vu Pham <vu@mellanox.com>

---
 drivers/infiniband/ulp/srp/ib_srp.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: ofed_kernel/drivers/infiniband/ulp/srp/ib_srp.c
===================================================================
--- ofed_kernel.orig/drivers/infiniband/ulp/srp/ib_srp.c
+++ ofed_kernel/drivers/infiniband/ulp/srp/ib_srp.c
@@ -1757,8 +1758,7 @@ static int srp_add_target(struct srp_hos
 
 	target->state = SRP_TARGET_LIVE;
 
-	scsi_scan_target(&target->scsi_host->shost_gendev,
-			 0, target->scsi_id, SCAN_WILD_CARD, 0);
+	scsi_scan_host(target->scsi_host);
 
 	return 0;
 }
